pandas/tests/frame/test_constructors.py

```diff
@@ -1726,10 +1726,16 @@ def test_constructor_with_datetimes(self):
         tm.assert_frame_equal(df, expected)
 
     def test_constructor_datetimes_with_nulls(self):
```

Comment:
can parameterize (later ok too)

Reply:
@jbrockmendel : do you plan on addressing this here or later?

Reply:
later